tests/integration/routes/account/patch-account-test.js

```diff
@@ -155,6 +254,21 @@ getServer(function (error, server) {
       })
     })
 
+    group.test('username change', function (t) {
+      var clock = lolex.install(0, ['Date'])
+      var couchdb = mockUsernameChange()
+      server.inject(usernameChangeOptions, function (response) {
+        clock.uninstall()
+
+        t.is(couchdb.pendingMocks()[0], undefined, 'all mocks satisfied')
+
+        t.is(response.statusCode, 204, 'returns 204 status')
+        t.is(response.result, null, 'returns no body')
```

Comment:
can we check for the `x-set-session` response header here as implemented in https://github.com/hoodiehq/hoodie-account-server/issues/80?